### PR TITLE
user properties in edit form

### PR DESCRIPTION
**Pull request by Nimoto**

### CODE REVIEW

File: src/actions/BackendEditUser.php [hunk 1]
```diff
@@ -78,6 +80,9 @@ public function run($id = null)
             if (true === $model->load($post) && true === $model->validate()) {
                 $attributes = $model->attributes;
                 unset($attributes['id']);
+                if (isset($post["User"]) && !empty($post["User"])) {
+                    $attributes = array_merge($attributes, $post["User"]);
```
Comment (bethrezen) [inline src/actions/BackendEditUser.php]: Please use single quotes if you don't need variables inside string

**Review by bethrezen — APPROVED** [review]
Approved with minor issue

File: src/actions/BackendEditUser.php [hunk 2]
```diff
@@ -84,6 +86,11 @@ public function run($id = null)
                 } else {
                     $user->scenario = $userClass::SCENARIO_PROFILE_UPDATE;
                     $res = $user->save();
+                    if (isset($post["User"]) && !empty($post["User"])) {
```
Comment (ivansal) [inline src/actions/BackendEditUser.php]: Change pls to single

File: src/actions/BackendEditUser.php [hunk 3]
```diff
@@ -84,6 +86,11 @@ public function run($id = null)
...
+                        $users = [User::find()->where(["id" => $user->id])->one()];
```
Comment (ivansal) [inline src/actions/BackendEditUser.php]: same

File: src/actions/BackendEditUser.php [hunk 4]
```diff
@@ -84,6 +86,11 @@ public function run($id = null)
...
+                        $users[0]->setAttributes($post["User"]);
```
Comment (ivansal) [inline src/actions/BackendEditUser.php]: same

**Review by ivansal — CHANGES_REQUESTED** [review]

**Pull request merged**